Create Popup window for mobile and desktop version

tjay1760 opened the pull request:
### In this Pull request
**I created a pop up window for mobile**
The popoup window displays additional details on the projects and includes two buttons to see the project live or the source on git hub.
**I created a popup for desktop sites**
The popoup window displays additional details on the projects and includes two buttons to see the project live or the source on git hub.
**Dynamic content generation**
The work cards for both the mobile and desktop version were generated dynamically using javascript

binhussen commented on styles.css:
```diff
@@ -22,19 +55,59 @@ body {
-    transition: all 0.3s;
+  .desktop-popup-img {
+    width: 100%;
+    height: 50%;
   }
 
-  img:hover {
-    transform: scale(1.1);
+  .popup-content-box {
+    display: grid;
+    grid-template-columns: 2fr 1fr;
+  }
```
- Great job for implementing popups 👍👍, I think You are losing some of the information from the popup detail because your popup is not scrollable, please make your popup scrollable to view all information 👍👍
- Kindly, the background behind the modal in the Figma design is blurred 👍👍 please make yours also fit the Figma design and have a better UI👍👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/218041171-22eeb59b-4c0d-44cd-b313-487b08becab8.png" width="600">|<img src="https://user-images.githubusercontent.com/52010789/218041152-bc331d9b-8057-440b-aad1-b55d416df4f9.png" width="600">|


😎️

🤝️

binhussen commented on styles.css:
```diff
@@ -454,415 +527,470 @@ body {
-    margin-top: 3%;
+  .mobile-popup {
+    height: 90%;
+    width: 80%;
+    background: #fff;
+    z-index: 2;
+    position: fixed;
+    top: 0;
+    border-radius: 16px;
+    border: 1px solid #dfe1e6;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    padding: 5%;
+    margin: 5%;
+    margin-top: 10%;
   }
 
-  .show {
-    display: block;
+  .popup-line {
+    width: 90%;
+    height: 2px;
+    background-color: #ebecf0;
+    margin: 5%;
   }
```
- Great job for implementing mobile popups 👍👍, I think it would be better if you make the background blur and update the `see live` and `see source` buttons to stick the Figma design and have better look👍👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/218041851-d5cdb884-6b2c-4cca-bd92-e33fe2647163.png" width="300">|<img src="https://user-images.githubusercontent.com/52010789/218041855-74427801-679c-48b5-b5e9-068d3a895904.png" width="300">|


😎️

🤝️

Gambit142 replied:
- [ ] Great work making some adjustments to the button. To make the buttons look identical, kindly add some padding to the _**See live**_ and _**See Source**_ buttons.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/218090961-361566cc-4570-4df1-99f8-5adfa303a9c3.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/218091039-e7cce4da-441a-4ec6-a28f-516853a46879.png)

binhussen reviewed (changes requested):
Hi @tjay1760 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<img src="https://media0.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif"/>

## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tjay1760 commented:
### The following changes were made to the repository
**Created a background blur for the popups** 
https://github.com/tjay1760/Mobile-potfolio/blob/0acbbe53d324d85f12124fbfd5bbd81dcae2d2b8/script.js#L531
https://github.com/tjay1760/Mobile-potfolio/blob/0acbbe53d324d85f12124fbfd5bbd81dcae2d2b8/script.js#L402
**Fixed scroll issue with desktop popup**
https://github.com/tjay1760/Mobile-potfolio/blob/0acbbe53d324d85f12124fbfd5bbd81dcae2d2b8/styles.css#L109
**Fixed buttons to match figma design**
https://github.com/tjay1760/Mobile-potfolio/blob/0acbbe53d324d85f12124fbfd5bbd81dcae2d2b8/styles.css#L601

Gambit142 commented on styles.css:
```diff
@@ -22,19 +59,68 @@ body {
-    background-color: #e5e5e5;
-    margin: 0;
-    padding: 0;
+  .desktop-popup {
+    position: fixed;
+    top: 0;
+    margin: 5%;
+    background-color: #fff;
+    z-index: 5;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 90%;
+    height: 100vh;
+    border-radius: 16px;
+    padding: 2%;
+    -webkit-backdrop-filter: blur(10px);
   }
```
- [ ] Great work making the popup scrollable. However, I noticed that the content and the popup itself are being cut off at the bottom. I would advise you to adjust it so that all contents can be seen. 💻 
- [ ] The buttons on the desktop view are not identical to that of the Figma template provided. Kindly make them identical as that is part of the requirement for this miles stone. 😎 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/218089803-a821db62-24c6-415b-a692-67438753fe69.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/218090095-8c7418de-475d-457a-a734-2f1c7a98cc7e.png)

### Your Design (Buttons on Desktop View)

![image](https://user-images.githubusercontent.com/77106962/218090259-95713ebb-ed40-40f1-beb4-593507d99ce3.png)

### Figm Design (Buttons on Desktop View)

![image](https://user-images.githubusercontent.com/77106962/218090364-87daa421-4c43-4c68-a361-2953aaa24a03.png)

Gambit142 commented on styles.css:
```diff
@@ -22,19 +59,68 @@ body {
+    overflow-y: scroll;
+  }
+
+  .popup-close {
+    position: absolute;
+    top: 3%;
+    right: 5%;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    color: #67798e;
+  }
```
- [OPTIONAL] Kindly consider adding this style rule `cursor: pointer;` to this selector. It would make the UI more interactive for users. 👍 😎

Gambit142 reviewed (changes requested):
Hi @tjay1760 @Munyabelden 👋, 

Great work on making the changes requested by a previous reviewer 👏 

## Highlights
- All linters checks passed ✔️ 
- Modal window opens when the button is clicked ✔️ 
- Closing icon removes modal window ✔️ 

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions

- Kindly consider the comment tagged _**[OPTIONAL]**_ under this review. 👍 😎 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tjay1760 commented:
Changes made to the project### 
Updated outdated and invalid entries in the readme file
Added a MIT license to the project
Fixed padding issues in the buttons to match the figma buttons
updated the desktop popup to allow all content to fit

AbayomiOlaoye commented on styles.css:
```diff
@@ -454,415 +560,478 @@ body {
-    float: right;
-    margin-right: 6.5%;
-    margin-top: 6.5%;
+  .popupbutton-child {
+    color: #396df2;
+    font-weight: 500;
+    font-size: 15px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid #396df2;
+    margin-top: 0%;
+    padding: 2.7%;
   }
```
- [ ] [OPTIONAL] :+1: You have done an amazing job implementing feedback from previous reviewers, and I think to make your project even more aesthetic and improve user experience for mobile users, it will be worth to make some small changes to improve the present UI. The buttons seem to big for the available space and it's into one another in a not so good way. Here's a screenshot to help you understand what I meant :point_down: 

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/218120689-f485f3d9-a80a-4ba5-b91a-cb9dbc15a10c.png)|![image](https://user-images.githubusercontent.com/99715640/218120790-9dc09e42-07f1-44f2-8fa3-73605c35c972.png)|

AbayomiOlaoye reviewed (approved):
Hi @tjay1760 :wave: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![applause](https://user-images.githubusercontent.com/99715640/213818870-485b6c36-1d87-4b93-9a0c-ce2ae86d9a9d.gif)

## Highlights
- Great implementation of reviewer's suggestions :heavy_check_mark: 
- Good UI implementation :heavy_check_mark: 
- Linter checks are passing :heavy_check_mark: 
- Good attitude to feedback :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
